doc/util/chpl2rst.py

```diff
@@ -229,6 +237,8 @@ def gen_rst(handle):
             codeline = ''.join(['    ', line])
             output.append(codeline)
 
+    if(islearnChapelInYMinutes and (not foundCommentExample)):
+        print('Warning: Failed to find special case of comment example in learnChapelInYMinutes.chpl')
```

Comment:
I Just made that change. I also added a little bit of sanity checking where if we find more than one such example, then that is also an error.

Reply:
:+1:  Note that the text of the OP should be adjusted to change "warning" to "error".  And then a few pro tips:

* If your OP contains the text "Resolves #<issue number>" it will link this PR and the issue you're resolving and auto-close the issue when this is merged (so that's a good practice)
* The DCO line isn't necessary in the OP of a PR like this (though if it's a single-commit branch when you open the PR, you'll typically get it by default).  There's no harm in including it, just wanted to point it out to prevent people from getting superstitious about it.